src/pages/publication/files_and_license_tab/FileCard.tsx

```diff
@@ -35,6 +37,13 @@ const StyledFileInfo = styled.div`
   justify-content: space-between;
 `;
 
+const StyledSelect = styled(TextField)`
+  .MuiSelect-root {
+    /* Ensure input height isn't expanded due to image content */
+    height: 1.1875rem;
```

Comment:
kan du definere dette som en const? MUI_INPUT_HEIGHT eller noe?

Reply:
Vet faktisk ikke... Kan man bruke en const i CSS? Må kanskje sendes som prop til Styled-Component i så fall?

Reply:
kanskje en spennende ting å finne ut på slutten på en fredag? :D

Reply:
Jodo, gikk det! Spørsmålet blir hva vi liker best.
```
const MUI_INPUT_HEIGHT = '1.1875rem';
const StyledSelect = styled(TextField)`
  .MuiSelect-root {
    height: ${MUI_INPUT_HEIGHT};
  }
`;
```
vs
```
const StyledSelect = styled(TextField)`
  .MuiSelect-root {
    /* Ensure input height isn't expanded due to image content */
    height: 1.1875rem;
  }
`;
```
Personlig liker jeg siste best, men jeg er kanskje mer glad i kommentarer enn resten? @pcb-unit @Eiriknil

Reply:
med mindre navnet på konstanten tilsier hva som er hensikten, kan du like godt bruke kommentar

Reply:
jeg ser for meg at det kan være nyttig å definere denne konstanten utenfor denne komponenten (fordi vi mest sannsynlig vil få bruk for den i fremtiden), men kan jo vente til da da